### PR TITLE
redid 409 (which also contains 408): Enhancement tMCR Vehicle Equipment profile Member, accepted driver permits

### PR DESCRIPTION
**Pull request by ue71603**
last two commits d57ef6a 7f18bb2 .Did overwrite the existing files.

fixes
#408  #409

### CODE REVIEW

File: xsd/NeTEx_publication.xsd [hunk 1]
```diff
@@ -7047,27 +7171,6 @@ Correct COnstraints for PointOnRoute
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
-		<!-- =====TypeOfFlexibleService============================== -->
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:7050]: Intentional?

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 2]
```diff
@@ -458,9 +474,79 @@ Rail transport, Roads and Road transport
 			<xsd:element name="Portable" type="xsd:boolean" minOccurs="0">
 				<xsd:annotation>
 					<xsd:documentation>Whether vehicle can be carried easily, e.g. scooter, skateboard, collapsible bicycle.
-</xsd:documentation>
+          </xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:477]: Remove newline.

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 3]
```diff
@@ -458,9 +474,79 @@ Rail transport, Roads and Road transport
...
+				<xsd:annotation>
+					<xsd:documentation>Types of Driver Licence that are accepted for Simple Vehicle Type.  +v2.0</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<!-- =======ACCEPTED DRIVER PERMIT ================================================= -->
+	<xsd:complexType name="acceptedDriverPermits_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a list ofACCEPTED DRIVER PERMITs.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:490]:
```suggestion
			<xsd:documentation>Type for a list of ACCEPTED DRIVER PERMITs.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 4]
```diff
@@ -458,9 +474,79 @@ Rail transport, Roads and Road transport
...
+					<xsd:annotation>
+						<xsd:documentation>Elements for an ACCEPTED DRIVER PERMIT.</xsd:documentation>
+					</xsd:annotation>
+				</xsd:group>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="AcceptedDriverPermitGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for a ACCEPTED DRIVER PERMIT.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:541]:
```suggestion
			<xsd:documentation>Elements for an ACCEPTED DRIVER PERMIT.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 5]
```diff
@@ -458,9 +474,79 @@ Rail transport, Roads and Road transport
...
+							<xsd:documentation>Identifier of VEHICLE EQUIPMENT PROILE.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:attribute>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="AcceptedDriverPermit_VersionStructure" abstract="false">
+		<xsd:annotation>
+			<xsd:documentation>Type for a ACCEPTED DRIVER PERMIT.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:527]:
```suggestion
			<xsd:documentation>Type for an ACCEPTED DRIVER PERMIT.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 6]
```diff
@@ -458,9 +474,79 @@ Rail transport, Roads and Road transport
...
+							<xsd:group ref="EntityInVersionGroup" minOccurs="0"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:group ref="AcceptedDriverPermitGroup"/>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="AcceptedDriverPermitIdType">
+						<xsd:annotation>
+							<xsd:documentation>Identifier of VEHICLE EQUIPMENT PROILE.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:518]:
```suggestion
							<xsd:documentation>Identifier of VEHICLE EQUIPMENT PROFILE.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 7]
```diff
@@ -458,9 +474,79 @@ Rail transport, Roads and Road transport
...
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="AcceptedDriverPermitGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for a ACCEPTED DRIVER PERMIT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="Name" type="MultilingualString" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Name of ACCEPTED DRIVER PERMIT.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:546]:
```suggestion
					<xsd:documentation>Name of an ACCEPTED DRIVER PERMIT.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 8]
```diff
@@ -973,7 +1059,7 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element name="RegistrationDate" type="xsd:date" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Dae of  registration or commissioning - may be used to determin age oif vehice</xsd:documentation>
+					<xsd:documentation>Dae of registration or commissioning - may be used to determin age oif vehice</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:1062]:
```suggestion
					<xsd:documentation>Date of registration or commissioning - may be used to determine age of the vehice.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 9]
```diff
@@ -1071,7 +1157,8 @@ Rail transport, Roads and Road transport
 	<!-- =======VEHICLE EQUIPMENT ================================================== -->
 	<xsd:element name="VehicleEquipmentProfile" substitutionGroup="DataManagedObject">
 		<xsd:annotation>
-			<xsd:documentation>Each instantiation of this ENTITY gives the number of items of one TYPE OF EQUIPMENT a VEHICLE MODEL should contain for a given PURPOSE OF EQUIPMENT PROFILE. The set of instantiations for one VEHICLE MODEL and one purpose gives one complete 'profile'.</xsd:documentation>
+			<xsd:documentation>Each instantiation of this ENTITY gives the number of items of one TYPE OF EQUIPMENT a VEHICLE MODEL should contain for a given PURPOSE OF EQUIPMENT PROFILE. The set of
+        instantiations for one VEHICLE MODEL and one purpose gives one complete 'profile'.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:1161]:
```suggestion
			<xsd:documentation>Each instantiation of this ENTITY gives the number of items of one TYPE OF EQUIPMENT a VEHICLE MODEL should contain for a given PURPOSE OF EQUIPMENT PROFILE. The set of instantiations for one VEHICLE MODEL and one purpose gives one complete 'profile'.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 10]
```diff
@@ -1132,12 +1219,89 @@ Rail transport, Roads and Road transport
...
+			<xsd:extension base="strictContainmentAggregationStructure">
+				<xsd:sequence>
+					<xsd:element ref="VehicleEquipmentProfileMember" maxOccurs="unbounded"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:element name="VehicleEquipmentProfileMember" substitutionGroup="VersionedChild">
+		<xsd:annotation>
+			<xsd:documentation>A element within a  VEHICLE EQUIPMENT PROFILE specifying the number of units of a given TYPE OF EQUIPMENT.
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:1244]:
```suggestion
			<xsd:documentation>A element within a VEHICLE EQUIPMENT PROFILE specifying the number of units of a given TYPE OF EQUIPMENT.
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 11]
```diff
@@ -1132,12 +1219,89 @@ Rail transport, Roads and Road transport
...
+</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:1245]:
```suggestion
			<xsd:documentation>A element within a VEHICLE EQUIPMENT PROFILE specifying the number of units of a given a TYPE OF EQUIPMENT.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 12]
```diff
@@ -1132,12 +1219,89 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element ref="TypeOfEquipmentRef" minOccurs="0"/>
 		</xsd:sequence>
 	</xsd:group>
 	<!-- ========= PURPOSE OF EQUIPMENT ============================================ -->
 	<xsd:element name="PurposeOfEquipmentProfile" abstract="false" substitutionGroup="TypeOfValue">
 		<xsd:annotation>
-			<xsd:documentation>A functional purpose which requires a certain set of EQUIPMENT of different types put together in a VEHICLE EQUIPMENT PROFILE or STOP POINT EQUIPMENT PROFILE.</xsd:documentation>
+			<xsd:documentation>A functional purpose which requires a certain set of EQUIPMENT of different types put together in a VEHICLE EQUIPMENT PROFILE or STOP POINT EQUIPMENT PROFILE. </xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:1304]:
```suggestion
			<xsd:documentation>A functional purpose which requires a certain set of EQUIPMENT of different types put together in a VEHICLE EQUIPMENT PROFILE or STOP POINT EQUIPMENT PROFILE.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 13]
```diff
@@ -1170,5 +1334,42 @@ Rail transport, Roads and Road transport
 			<xsd:extension base="TypeOfValue_VersionStructure"/>
 		</xsd:complexContent>
 	</xsd:complexType>
+	<!-- =========TYPE OF DRIVER PERMIT. ============================================ -->
+	<xsd:element name="TypeOfDriverPermit" abstract="false" substitutionGroup="TypeOfValue">
+		<xsd:annotation>
+			<xsd:documentation>A type of driving license (e.g. https://en.wikipedia.org/wiki/European_driving_licenceA type of driving license (e.g. https://en.wikipedia.org/wiki/European_driving_licence).",  +V2.0
+</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:1341]:
```suggestion
			<xsd:documentation>A type of driving license (e.g. https://en.wikipedia.org/wiki/European_driving_licenceA type of driving license (e.g. https://en.wikipedia.org/wiki/European_driving_licence).",  +V2.0</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd (hunk 2 above)
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:477]: Remove new line.

File: xsd/NeTEx_publication.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/NeTEx_publication.xsd:7050]: Was duplicate. So I guess yes. But this is something Nick and Christophe should check..

**Review by Aurige — APPROVED** [review]
include several things (drivers permit, capacity Id, description fix..) but Ok for me

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 14]
```diff
@@ -961,7 +1047,7 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Short Name of VEHICLE.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="Description" type="MultilingualString" minOccurs="0">
+			<xsd:element name="Description" minOccurs="0">
```
Comment (trurlurl) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:1050]:
```suggestion
			<xsd:element name="Description" type="MultilingualString" minOccurs="0">
```

File: examples/functions/newModes/NewModes-CycleSharingExample.xml [hunk 15]
```diff
@@ -117,12 +120,26 @@
...
+								<TypeOfEquipment version="any" id="charger">
+									<Name>Charger</Name>
+								</TypeOfEquipment>
+							</values>
+						</ValueSet>
+						<ValueSet version="any" id="DriverPermits" classOfValues="TypeOfDriverPermit">
+							<Name>Driving licence types</Name>
+							<values>
+								<TypeOfDriverPermit version="any" id="ecycle_permit">
+									<Name>ecycle </Name>
```
Comment (skinkie) [inline examples/functions/newModes/NewModes-CycleSharingExample.xml:138]:
```suggestion
									<Name>ecycle</Name>
```

File: examples/functions/newModes/NewModes-CycleSharingExample.xml [hunk 16]
```diff
@@ -117,12 +120,26 @@
...
+								</TypeOfDriverPermit>
+								<TypeOfDriverPermit version="any" id="car_permit">
+									<Name>car </Name>
```
Comment (skinkie) [inline examples/functions/newModes/NewModes-CycleSharingExample.xml:141]:
```suggestion
									<Name>car</Name>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_support.xsd [hunk 17]
```diff
@@ -449,7 +457,7 @@ Rail transport, Roads and Road transport
 	</xsd:simpleType>
 	<xsd:simpleType name="PropulsionTypeEnumeration">
 		<xsd:annotation>
-			<xsd:documentation>Allowed values for type of power  +v1/2/2</xsd:documentation>
+			<xsd:documentation>Allowed values for type of power +v1/2/2</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_support.xsd:460]:
```suggestion
			<xsd:documentation>Allowed values for type of power +v1.2.2</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_support.xsd [hunk 18]
```diff
@@ -483,7 +491,7 @@ Rail transport, Roads and Road transport
 	</xsd:simpleType>
 	<xsd:simpleType name="SimpleVehicleCategoryEnumeration">
 		<xsd:annotation>
-			<xsd:documentation>Allowed values for Personal Vehicle Categories  _v1.2.2</xsd:documentation>
+			<xsd:documentation>Allowed values for Personal Vehicle Categories _v1.2.2</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_support.xsd:494]:
```suggestion
			<xsd:documentation>Allowed values for Personal Vehicle Categories +v1.2.2</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 19]
```diff
@@ -593,7 +679,7 @@ Rail transport, Roads and Road transport
 							<xsd:group ref="PassengerCapacityGroup"/>
 						</xsd:sequence>
 					</xsd:sequence>
-					<xsd:attribute name="id" type="PassengerCapacityIdType" use="required">
+					<xsd:attribute name="id" type="PassengerCapacityIdType" use="optional">
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:682]: Why optional?

Comment (ue71603) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:682]:
```suggestion
					<xsd:attribute name="id" type="PassengerCapacityIdType" use="required">
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd (hunk 2 above)
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd]: done

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd (hunk 19 above)
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd]: mandatory now. But not done by me.

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 20]
```diff
@@ -457,12 +473,81 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element name="Portable" type="xsd:boolean" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Whether vehicle can be carried easily, e.g. scooter, skateboard, collapsible bicycle.
-</xsd:documentation>
+					<xsd:documentation>Whether vehicle can be carried easily, e.g. scooter, skateboard, collapsible bicycle.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:476]:
```suggestion
					<xsd:documentation>Whether vehicle can be carried easily, e.g. scooter, skateboard, foldable bicycle.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 21]
```diff
@@ -457,12 +473,81 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="acceptedDriverPermits" type="acceptedDriverPermits_RelStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Types of Driver Licence that are accepted for Simple Vehicle Type.  +v2.0</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:481]: Do the uppercases make sense?

File: examples/functions/newModes/NewModes-CycleSharingExample.xml [hunk 22]
```diff
@@ -19,15 +19,18 @@
 							  Deposit required.
 						   SALES PACKAGES  (a) mobile app for preregistered users only.
 
-(C) CEN 2021
+     Update
+          2023.01.30 Add AcceptedDRiverPermit and VehicleEquipmentProifle exampls
```
Comment (trurlurl) [inline examples/functions/newModes/NewModes-CycleSharingExample.xml:23]:
```suggestion
          2023.01.30 Add AcceptedDriverPermit and VehicleEquipmentProfile examples
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 23]
```diff
@@ -457,12 +473,81 @@ Rail transport, Roads and Road transport
...
 			<xsd:element name="Portable" type="xsd:boolean" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Whether vehicle can be carried easily, e.g. scooter, skateboard, collapsible bicycle.
-</xsd:documentation>
+					<xsd:documentation>Whether vehicle can be carried easily, e.g. scooter, skateboard, foldable bicycle.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="acceptedDriverPermits" type="acceptedDriverPermits_RelStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Types of Driver Licence that are accepted for Simple Vehicle Type.  +v2.0</xsd:documentation>
```
Comment (ue71603) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:481]:
```suggestion
					<xsd:documentation>Types of driver licence that are accepted for simple vehicle type.  +v2.0</xsd:documentation>
```

**Comment by ue71603** [comment]
@skinkie  if ok for you too, then we merge.

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd (hunk 21 above)
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd]: changed

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 24]
```diff
@@ -201,7 +217,7 @@ Rail transport, Roads and Road transport
 				</xsd:element>
 				<xsd:element name="TypeOfFuel" type="FuelTypeEnumeration" minOccurs="0">
 					<xsd:annotation>
-						<xsd:documentation>The type of fuel used by a vehicle of the type. DEPRECATED NAME  1.2.2</xsd:documentation>
+						<xsd:documentation>The type of fuel used by a vehicle of the type. DEPRECATED NAME 1.2.2</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:220]:
```suggestion
						<xsd:documentation>The type of fuel used by a vehicle of the type. DEPRECATED NAME v1.2.2</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd [hunk 25]
```diff
@@ -1170,5 +1331,41 @@ Rail transport, Roads and Road transport
...
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="TypeOfValueGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="TypeOfDriverPermitIdType">
+						<xsd:annotation>
+							<xsd:documentation>Identifier of PURPOSE OF EQUIMENT PROFILE.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd:1355]:
```suggestion
							<xsd:documentation>Identifier of PURPOSE OF EQUIPMENT PROFILE.</xsd:documentation>
```

**Review by skinkie — APPROVED** [review]

**Comment by ue71603** [comment]
@Aurige  when ok for you, then we can merge.

**Pull request merged**